### PR TITLE
Reenable sharedfx with arm64 queue skipped

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by HaoK** [comment]
Reenables the shared fx tests, but skips them on helix arm since these tests don't actually test anything that varies on platform

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -716,7 +716,7 @@ stages:
   # Helix ARM64
   - template: jobs/default-build.yml
     parameters:
-      condition: and(eq(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'))
+      #condition: and(eq(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'))
```
Comment (dougbu) [inline .azure/pipelines/ci.yml:719]: Temporary. correct?

File: src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj [hunk 2]
```diff
@@ -3,9 +3,10 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore</RootNamespace>
-    <!-- https://github.com/dotnet/aspnetcore/issues/7939: This unit test requires the shared framework be available in Helix. -->
-    <BuildHelixPayload>false</BuildHelixPayload>
     <VerifyAncmBinary Condition="'$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm'">true</VerifyAncmBinary>
+    <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
+    <TestDependsOnAspNetRef>true</TestDependsOnAspNetRef>
+    <SkipHelixArm>true</SkipHelixArm>
```
Comment (dougbu) [inline src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj:9]: To confirm: This is the _only_ change from the prior PR reneabling the SharedFx tests?

**Review by dougbu — APPROVED** [review]
Needs the #20669 fix too, correct? (Why isn't the current runtests.sh file causing problems in this PR? Does the assignment work when the value is `0`?)

File: src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj (hunk 2 above)
Comment (HaoK) [reply src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj:9]: Yep, I'll add the fix runtests fix and back out the extra queues

**Pull request merged**